ember-flight-icons/tests/dummy/app/styles/app.css

```diff
@@ -60,46 +65,67 @@ h2.ds-h2 {
 }
 .ds-form-group-inline input.ds-input:focus {
   border: 1px solid var(--brand);
+  box-shadow: 0 0 0 3px var(--info-l1);
+  outline: none;
 }
 
 ul.ds-ul-grid {
   display: grid;
   font-family: var(--font-body);
-  gap: 10px;
+  gap: 16px;
   grid-auto-flow: dense;
-  grid-auto-rows: 125px;
-  grid-template-columns: repeat(auto-fill, minmax(125px, 1fr));
+  grid-auto-rows: 148px;
+  grid-template-columns: repeat(auto-fill, minmax(148px, 1fr));
   list-style: none;
   margin: 0;
   padding: 0;
 }
 ul.ds-ul-grid li.ds-li {
   align-items: center;
-  border: 1px solid black;
   display: flex;
   flex-direction: column;
-  font-size: 0.85em;
+  font-size: 0.75em;
   gap: 1ch;
   justify-content: center;
   padding: 0.5em 0.15em 0.15em;
+  background: var(--gray-7);
+  border-radius: 3px;
+  box-shadow: 0 0 0 1px var(--gray-5);
 }
 ul.ds-ul-grid li.ds-li:hover {
-  background-color: aliceblue;
+  background-color: var(--info-l2);
+  box-shadow: 0 0 0 1px var(--info-l1);
 }
-ul.ds-ul-grid li.ds-li svg.flight-icon {
+ul.ds-ul-grid li.ds-li .icon-frame {
```

Comment:
@dizzyup I have been trying to prefix all classes in the demo app with `.ds-` to get into the practice, that way we avoid naming collisions with proper components in the future.

Reply:
👍 Fixed!

I was going to look at moving it to tailwind but thought better of it and decided to save that bigger discussion for another day 😉